app/controllers/admin/permissions/system-roles.js

```diff
@@ -8,6 +8,19 @@ export default Controller.extend({
     },
     addSystemRole() {
       this.set('isAddSystemRoleModalOpen', false);
+    },
+    updatePermissions() {
+      this.set('isLoading', true);
+      let userPermissions = this.get('model');
+      userPermissions.save()
+        .then(() => {
+          this.set('isLoading', false);
+          this.notify.success(this.l10n.t('User permissions have been saved successfully.'));
+        })
+        .catch(()=> {
+          this.set('isLoading', false);
+          this.notify.error(this.l10n.t('An unexpected error has occured. User permissions not saved.'));
```

Comment:
@CosmicCoder96 Can you please update the message of occured? The correct one is 'occurred'. :)